doc/devices/remote.rst

```diff
@@ -0,0 +1,78 @@
+The Remote device
+=================
+
+Pennylane's Remote device gives access to remote backends from Strawberry Fields including
+`hardware backends <https://strawberryfields.ai/photonics/hardware/index.html>`_.
+
+Accounts and Tokens
+~~~~~~~~~~~~~~~~~~~
+
+By default, the ``strawberryfields.ai`` device will attempt to use an already active or stored
+Strawberry Fields account. If the device finds no account it will raise a warning:
+
+.. code::
+
+    'WARNING:strawberryfields.configuration:No Strawberry Fields configuration file found.'
+
+You can use the ``strawberryfields.store_account("<my_token>")`` function to
+permanently store an account.  Alternatively, you can use the `Strawberry
+Fields command line interface for configuration
+<https://strawberryfields.readthedocs.io/en/stable/code/sf_cli.html>`__.
+
+.. warning:: Never publish code containing your token online.
+
+Usage
+~~~~~
+
+You can instantiate the Remote device in PennyLane as follows:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    dev = qml.device('strawberryfields.ai', backend="X8", wires=8, shots=10, sf_token="XXX")
```

Comment:
Minor question, but do we need to specify the number of wires if using a physical backend? Maybe we could just infer it from SF.

Reply:
Something that comes to mind is the forest plugin, where `wires` was not required for physical lattices

Reply:
Valid point! I've included logic to use circuitspecs from SF